content.md

```diff
@@ -269,9 +327,60 @@ This is a simple VS Code extension that allows you to comment out your ERB files
 
 The Ruby LSP is an extension that provides performant rich features for Ruby. It connects to the ruby-lsp language server gem to analyze Ruby code and enhance the user experience
 
+## Quiz
+
+- What is the purpose of the `rbenv` tool in Ruby development?
+
+  - It is used to manage different versions of the Ruby programming language on a single machine.
+    - Correct!
+  - It is used to install Ruby on Rails.
+    - Not correct. `rbenv` is a version manager for Ruby, not a Rails installer.
+  - It is a package manager for macOS.
+    - Not correct. `rbenv` is a version manager for Ruby, not a package manager for macOS.
+  - It is a version control system.
+    - Not correct. `rbenv` is a version manager for Ruby, not a version control system.
+      {: .choose_best #rbenv title="What is the Purpose of rbenv" points="1" answer="1"}
+
+- How can you verify that Rails has been installed successfully?
+
+  - Run `brew install rails`
+    - Not correct. This command is used to install Rails via Homebrew, not to verify its installation.
+  - Run `gem install rails`
+    - Not correct. This command is used to install Rails via RubyGems, not to verify its installation.
+  - Run `rails -v`
+    - Correct!
+  - Run `rbenv init`
+    - Not correct. This command is used to initialize `rbenv`, not to verify Rails installation.
+      {: .choose_best #rails title="How Can You Verify Rails Install?" points="1" answer="3"}
+
+- Which command is used to set the default Ruby version for the machine?
+
+  - `rbenv install 3.2.1`
+    - Not correct. This command is used to install a specific Ruby version via `rbenv`, not to set the default Ruby version.
+  - `rbenv local 3.2.1`
+    - Not correct. This command is used to set the local Ruby version for a specific project, not to set the default Ruby version.
+  - `rbenv global 3.2.1`
+    - Correct!
+  - `ruby -v`
+    - Not correct. This command is used to check the current Ruby version, not to set the default Ruby version.
+      {: .choose_best #default-ruby title="Command for default Ruby on sysytem?" points="1" answer="3"}
+
+- What is the Ruby LSP extension used for?
+
+  - To provide performant rich features for Python
+    - Not correct. The Ruby LSP extension is for Ruby, not Python.
+  - To install Ruby on Rails
+    - Not correct. The Ruby LSP extension is for enhancing Ruby development, not for installing Rails.
+  - To set up PostgreSQL
+    - Not correct. The Ruby LSP extension is for enhancing Ruby development, not for setting up PostgreSQL.
+  - To provide performant rich features for Ruby
```

Comment:
can we provide examples of "performant rich features"? eg Go to definition, Showing documentaton on hover, Autocomplete, Semantic highlighting, etc.